openpower/package/openpower-pnor/openpower-pnor.mk

```diff
@@ -151,6 +153,7 @@ define OPENPOWER_PNOR_INSTALL_IMAGES_CMDS
             -targeting_RW_binary_filename $(BR2_OPENPOWER_TARGETING_ECC_FILENAME).unprotected \
             -wofdata_binary_filename $(OPENPOWER_PNOR_SCRATCH_DIR)/$(BR2_WOFDATA_BINARY_FILENAME) \
             -memddata_binary_filename $(OPENPOWER_PNOR_SCRATCH_DIR)/$(BR2_MEMDDATA_BINARY_FILENAME) \
+            -ocmbfw_binary_filename $(OPENPOWER_PNOR_SCRATCH_DIR)/$(BR2_OCMBFW_PROCESSED_FILENAME) \
```

Comment:
This works even for the xml layouts without this section, right?

Reply:
Yes, there is a default value that will be used for BR2_OCMBFW_PROCESSED_FILENAME and in the tools themselves nothing is done with this parameter unless the layout file actually has an OCMBFW partition. If a system doesn't have said partition, nothing would be done.

Reply:
agreed - seen this in action with other pnor makefile changes